src/probability/independence.lean

```diff
@@ -334,13 +348,117 @@ end
 
 variables {m0 : measurable_space Ω} {μ : measure Ω}
 
+-- todo move
+@[simp] lemma mem_ite_univ_right {ι} (p : Prop) [decidable p] (t : set ι) (x : ι) :
+  x ∈ ite p t set.univ ↔ (p → x ∈ t) :=
+by split_ifs; simp [h]
+
+-- todo move
+@[simp] lemma mem_ite_univ_left {ι} (p : Prop) [decidable p] (t : set ι) (x : ι) :
+  x ∈ ite p set.univ t ↔ (¬ p → x ∈ t) :=
+by split_ifs; simp [h]
```

Comment:
These two lemmas should be moved, but I have no idea of where they should go. Any suggestion?

Reply:
@YaelDillies said you should put it in `data.set.basic`. They also said you should add a `dite` and a `not` version

Reply:
Seems like the `not` version already exists! I had only seen `mem_ite_univ_right`.

Reply:
Thank you! I'll look at that file.